Source/Core/DolphinQt/FIFO/FIFOAnalyzer.cpp

```diff
@@ -478,14 +478,13 @@ void FIFOAnalyzer::UpdateDescription()
   QString text;
   if (*cmddata == OpcodeDecoder::GX_LOAD_BP_REG)
   {
-    std::string name;
-    std::string desc;
-    GetBPRegInfo(cmddata + 1, &name, &desc);
+    const u8 cmd = *(cmddata + 1);
+    const u32 value = Common::swap32(cmddata + 1) & 0xFFFFFF;
+    auto [name, desc] = GetBPRegInfo(cmd, value);
```

Comment:
Should be const.  Same for previous commit (175e9a02685c5a67f4309d45aedc4895e0f4797c, which I think should be merged with this one, and I'm not sure actually compiles when it's before this).  Also should probably use `Common::swap24` here.